osquery/logger/plugins/aws_kinesis.h

```diff
@@ -47,6 +48,7 @@ class KinesisLogForwarder : public BufferedLogForwarder {
   std::shared_ptr<Aws::Kinesis::KinesisClient> client_{nullptr};
 
   FRIEND_TEST(KinesisTests, test_send);
+  int retry_count = FLAGS_aws_kinesis_failed_upload_retry_count;
```

Comment:
This should be a `size_t`. It should also be initialized during `setUp` as the config may want to change the flag value. Right now it will only every be the default value.